add note to enable CentOS firewall (#8606)
wazuh/wazuh-documentation: source/_templates/installations/common/firewall-ports-note.rst

> Note that firewalls can block communication between Wazuh components on different hosts. Refer to the :ref:`Required ports <default_ports>` section and ensure the necessary ports are open.

Author: Maybe it will be better to use a `note` tag here 
@javimed

> .. note::
>    
>    CentOS systems have a firewall enabled by default that may block communication between Wazuh components if they are spread across different hosts. Ensure that the necessary ports are open in your firewall configuration. For a complete list of required ports, refer to the :ref:`Required ports <default_ports>` section in the architecture documentation.

Reviewer: ```suggestion
Note that firewalls can block communication between Wazuh components on different hosts. Refer to the :ref:`Required ports <default_ports>` section and ensure the necessary ports are open.
```

Reviewer: I would add as few notes as possible. I think adding it as a paragraph can work well.
